Trendyol.App.WebApi/Trendyol.App.WebApi.csproj

```diff
@@ -133,6 +133,8 @@
     <Compile Include="DeepLogging\IDeepLogger.cs" />
     <Compile Include="DeepLogging\SqlDeepLogger.cs" />
     <Compile Include="Enums\DataType.cs" />
+    <Compile Include="Filters\BasicAuthenticationFilter.cs" />
+    <Compile Include="Filters\IAuthenticationChecker.cs" />
```

Comment:
This is not the right place for this interface.
It needs to be placed in a folder named "Authentication" at project level.